crates/router/src/connector/adyen/transformers.rs

```diff
@@ -929,20 +941,30 @@ impl<'a> TryFrom<&api_models::payments::BankDebitData> for AdyenPaymentMethod<'a
                 account_number,
                 routing_number,
                 billing_details: _,
-                bank_account_holder_name,
+                card_holder_name,
+                bank_account_holder_name: _,
```

Comment:
```suggestion
                card_holder_name,
                ..
```

You can use this destructuring format since the other fields aren't read. This is an optional change, does not change any code behavior.